src/components/UseCaseDialog.jsx

```diff
@@ -1,32 +1,26 @@
 import React from 'react'
 import { translate } from '../plugins/i18n'
-
 import ConnectorList from './ConnectorList'
-import Dialog from './Dialog'
+import Modal from 'cozy-ui/react/Modal'
+import ModalContent from 'cozy-ui/react/Modal/Content'
```

Comment:
You know what sounds complicated ?

CSS Modules.

CSS Modules sounds complicated.